env.d/development/common.dist

```diff
@@ -53,3 +53,6 @@ JOANIE_BACKOFFICE_BASE_URL="http://localhost:8072"
 
 # must be reachable for signature email notifications
 DEVELOPER_EMAIL="developer@example.com"
+
+# Security for remote endpoints API
+JOANIE_AUTHORIZED_API_TOKENS = []
```

Comment:
According [to the documentation](https://django-configurations.readthedocs.io/en/stable/values.html#configurations.values.ListValue), this environment variable should be written
```suggestion
JOANIE_AUTHORIZED_API_TOKENS = "first item, second item"
```